scripts/restore_anonymised_db.sh

```diff
@@ -4,20 +4,63 @@
 # ./scripts/restore_anonymised_db.sh [branch-name]
 # ./scripts/restore_anonymised_db.sh ap-2555
 
-echo "Finding pod for branch $1 in UAT"
-POD=$(kubectl -n laa-apply-for-legalaid-uat get pods | grep -o -m4 "apply-$1-.*2/2" | head -n1 | cut -d' ' -f 1)
-echo "$POD"
+production_warning() {
+  echo "
+  -------------------------------------------
+  THIS SCRIPT IS NOT TO BE USED ON PRODUCTION
+  -------------------------------------------
+"
+}
 
-echo "Copying anonymised db to kubernetes pod..."
-kubectl -n laa-apply-for-legalaid-uat cp --retries=10 ./tmp/production.anon.sql -c web laa-apply-for-legalaid-uat/"$POD":./tmp/anonymised_db.sql
+help_msg() {
+  echo "Usage: db_export.sh [environment]
+  Where:
+  environment  [staging or a branch name, i.e. ap-1234]"
+  exit 1
+}
+
+case "$1" in
+"production")
+  production_warning
+  help_msg
+  ;;
+"")
+  help_msg
+  ;;
+"staging")
+  environment=$1
+  echo "Finding pod in $environment"
+  POD=$(kubectl -n "laa-apply-for-legalaid-$environment" get pods | grep -o -m4 "apply-for-.*2/2" | tail -n1 | cut -d' ' -f 1)
+  ;;
+*)
+  environment='uat'
+  echo "Finding pod for branch $1 in $environment"
+#  POD=$(kubectl -n laa-apply-for-legalaid-$environment get pods | grep -o -m4 "apply-$1-.*2/2" | head -n1 | cut -d' ' -f 1)
```

Comment:
nit: is there a purpose for leaving this comment, considering we could copy and paste L33 for UAT?

Reply:
Nope, that was a mistake, line 33 has a slightly different grep command.
I was restoring to staging so missed this!